package/harvester-os/files/etc/systemd/system/rke2-shutdown.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Kill RKE2 Containers for clean unmount
+DefaultDependencies=no
+RefuseManualStart=yes
+Before=reboot.target halt.target shutdown.target poweroff.target umount.target final.target
+
+[Install]
+RequiredBy=reboot.target halt.target shutdown.target poweroff.target umount.target final.target
```

Comment:
do we need the service to be active in the `umount.target final.target` stage?

Reply:
I tried it and it does work.